pkg/tfbridge/detailed_diff.go

```diff
@@ -474,6 +474,16 @@ func (differ detailedDiffer) makeDetailedDiffPropertyMap(
 	return result
 }
 
+func MakeDetailedDiffV2(
```

Comment:
Mark in a doc comment that this is internal. This Go package has been getting very large but it's been difficult to break it up a bit. It does however contain a mix of user-facing and internal defs like this one.